.github/workflows/performance.yml

```
@@ -123,5 +123,19 @@ jobs:
       - name: Print Performance tests results
         run: "node ./tests/performance/results.js"
 
+      - uses: actions/github-script@98814c53be79b1d30f795b907e553d8679345975 # v6.4.0
+        id: commit-timestamp
+        with:
+            github-token: ${{secrets.GITHUB_TOKEN}}
+            script: |
+                const commit_details = await github.rest.git.getCommit({owner: context.repo.owner, repo: context.repo.repo, commit_sha: context.sha});
+                return parseInt((new Date( commit_details.data.author.date ).getTime() / 1000).toFixed(0))
+      - name: Publish performance results
```

Nit-pick: Maybe separate the group from the others with a blank line, like we're doing elsewhere.
```suggestion

      - name: Publish performance results
```

---

Great catch.